cypress/integration/business-pages/business-pages.test.js

```diff
@@ -60,7 +60,7 @@ describe('Business pages test ', () => {
     cy.get('[data-cy=en]').should('be.visible');
     cy.get('[data-cy=en]').contains('en');
     cy.get('[data-cy=page-header-ua]').should('be.visible');
-    cy.get('[data-cy=page-header-ua]').contains('Заголовок uk');
+    cy.get('[data-cy=page-header-ua]').contains('Заголовок ua');
```

Comment:
Compare this cypress element with appropriate title in config file (it helps us to check whether this title shows right or not